### ISSUE TITLE
Query string and struct with single field named `v` -> Error: expected &str, found enum `std::option::Option`

### ISSUE THREAD

**Issue by rofrol** [SergioBenitez/Rocket#265]
1. The version of Rocket you're using. Ensure it's the latest, if possible.
0.2.5
  2. A brief description of the bug that includes:
    * The nature of the bug.
    * When the bug occurs.
      When struct has one and only one field named `v`.
    * What you expected vs. what actually happened.
       It compiles vs error.
  3. How you discovered the bug. Short test cases are especially useful.

Code:

```rust
#![feature(plugin, custom_derive)]
#![plugin(rocket_codegen)]

extern crate rocket;

#[macro_use]
extern crate rocket_contrib;

use rocket_contrib::{JSON, Value};

#[derive(FromForm)]
struct Person<'r> {
    v: &'r str,
}

#[get("/file?<person>")]
fn styles_with_query(person: Person) -> String {
    format!("Hello {}!", person.v)
}

#[error(404)]
fn not_found() -> JSON<Value> {
    JSON(json!({
        "status": "error",
        "reason": "Resource was not found."
    }))
}

fn main() {
    rocket::ignite()
        .mount("/", routes![styles_with_query])
        .catch(errors![not_found])
        .launch();
}
```
Running:
```shell
$ rustup install nightly-2017-04-16
$ rustup override set nightly-2017-04-16
$ cargo build
   Compiling bug v0.1.0 (file:///C:/projects/rust/rocket-bug)
error[E0308]: match arms have incompatible types
  --> src\main.rs:11:10
   |
11 | #[derive(FromForm)]
   |          ^^^^^^^^ expected &str, found enum `std::option::Option`
   |
   = note: expected type `&str`
              found type `std::option::Option<_>`
note: match arm with an incompatible type
  --> src\main.rs:11:10
   |
11 | #[derive(FromForm)]
   |          ^^^^^^^^

error: aborting due to previous error

error: Could not compile `rocket-bug`.

To learn more, run the command again with --verbose.
```
  4. Ideas, if any, about what Rocket is doing incorrectly.
   It works if field has any other name than `v`.

**Comment by SergioBenitez** [comment]
Confirmed! Fix is incoming.

**Comment by SergioBenitez** [comment]
The fix is in v0.2.6.

**Comment by rofrol** [comment]
Maybe there should be specific error message in error catalog when someone somehow encounters this: "Don't use __v, __e, __k for field name" etc.
